pootle/apps/pootle_translationproject/views.py

```diff
@@ -128,16 +128,13 @@ def browse(request, translation_project, dir_path, filename=None):
     ctx.update(get_browser_context(request))
 
     # TODO improve plugin logic
-    if "import_export" in settings.INSTALLED_APPS and request.user.is_authenticated():
+    can_upload = (not translation_project.is_terminology_project and
+                  (check_permission('translate', request)))
+    if "import_export" in settings.INSTALLED_APPS and can_upload:
         from import_export.views import handle_upload_form
-
         ctx.update(handle_upload_form(request))
-
-        has_download = (not translation_project.is_terminology_project and
-                        (check_permission('translate', request) or
-                         check_permission('suggest', request)))
         ctx.update({
-            'display_download': has_download,
+            'display_download': True,
```

Comment:
I think we might want to allow all authenticated users to download the translation files.